server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/QualityGatesWsSupport.java

```diff
@@ -49,10 +56,32 @@ QualityGateConditionDto getCondition(DbSession dbSession, long id) {
     return checkFound(dbClient.gateConditionDao().selectById(id, dbSession), "No quality gate condition with id '%d'", id);
   }
 
+  /**
+   * @deprecated use {@link #isQualityGateAdmin(OrganizationDto)} instead
+   */
+  @Deprecated
   boolean isQualityGateAdmin() {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)